src/objectives/distance_utils.jl

```diff
@@ -0,0 +1,69 @@
+"""
+    distance(ce::AbstractCounterfactualExplanation, p::Real=2)
+
+Computes the distance of the counterfactual to the original factual.
+"""
+function distance(
+    ce::AbstractCounterfactualExplanation;
+    from::Union{Nothing,AbstractArray}=nothing,
+    agg=mean,
+    p::Real=1,
+    weights::Union{Nothing,AbstractArray}=nothing,
+)
+    if isnothing(from)
+        from = CounterfactualExplanations.factual(ce)
+    end
+    x = CounterfactualExplanations.factual(ce)
```

Comment:
We're not using this x variable anywhere so it could be removed.

Reply:
Good point